backend/census_historical_migration/workbooklib/secondary_auditors.py

```diff
@@ -59,6 +64,30 @@
 ]
 
 
+def xform_address_state(secondary_auditors):
+    """Default missing address_state to GSA_MIGRATION"""
+    change_records = []
+    is_empty_address_state_found = False
+
+    for secondary_auditor in secondary_auditors:
+        address_state = string_to_string(secondary_auditor.CPASTATE)
+        if not address_state:
```

Comment:
We want to call `track_transformations` regardless of `if not address_state` condition. However, we want save the record (i.e., `InspectionRecord.append_secondary_auditor_changes(change_records)`) only if is_empty_address_state_found` is true.  @phildominguez-gsa